src/decision_makers/kick_decision_maker.py

```diff
@@ -10,10 +10,11 @@
 class KickDecisionMaker(IDecisionMaker):
     def __init__(self):
         self.bhv_kick_planner = BhvKickPlanner()
-        # self.bhv_kick_planner = BhvStarterKickPlanner()
+        self.bhv_kick_planner = BhvStarterKickPlanner()
```

Comment:
for default we should not use Starter classes, but we can have a variable in sample player agent, for example starter=true/false and we can choose planner or starter planner based on that variable